.travis.yml

```diff
@@ -19,4 +19,5 @@ jobs:
       os: linux
       script:
         - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate();'
+        - travis_wait 20 julia --project=docs/ docs/make.jl
```

Comment:
I suggest to only use `travis_wait` if necessary. It hides the output log, which may contain debug information.

Reply:
👍  
i removed `travis_wait` in master https://github.com/JuliaReach/RangeEnclosures.jl/blob/master/.travis.yml#L22